my-recipe-collection-index.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
```

Comment:
You should now rename this file to be `index.php` - index is a special filename that indicates that this is the homepage of the site